src/transaction/hooks/useWriteContract.ts

```diff
@@ -13,42 +13,30 @@ export function useWriteContract({
   setLifeCycleStatus,
   transactionHashList,
 }: UseWriteContractParams) {
-  try {
-    const { status, writeContractAsync, data } = useWriteContractWagmi({
-      mutation: {
-        onError: (e) => {
-          const errorMessage = isUserRejectedRequestError(e)
-            ? 'Request denied.'
-            : GENERIC_ERROR_MESSAGE;
-          setLifeCycleStatus({
-            statusName: 'error',
-            statusData: {
-              code: 'TmUWCh01', // Transaction module UseWriteContract hook 01 error
-              error: e.message,
-              message: errorMessage,
-            },
-          });
-        },
-        onSuccess: (hash: Address) => {
-          setLifeCycleStatus({
-            statusName: 'transactionLegacyExecuted',
-            statusData: {
-              transactionHashList: [...transactionHashList, hash],
-            },
-          });
-        },
+  const { status, writeContractAsync, data } = useWriteContractWagmi({
+    mutation: {
+      onError: (e) => {
+        const errorMessage = isUserRejectedRequestError(e)
+          ? 'Request denied.'
+          : GENERIC_ERROR_MESSAGE;
+        setLifeCycleStatus({
+          statusName: 'error',
+          statusData: {
+            code: 'TmUWCh01', // Transaction module UseWriteContract hook 01 error
+            error: e.message,
+            message: errorMessage,
+          },
+        });
       },
-    });
-    return { status, writeContractAsync, data };
-  } catch (err) {
```

Comment:
So, I was reflecting on when this `try/catch` will fire off, and I couldn't figure out a real case this can go off.

So I removed it for now.